environment.yml

```diff
@@ -1,4 +1,4 @@
-name: perceiver-io
+name: model-io
```

Comment:
Keep environment name `perceiver-io`?

Reply:
I would say that model io is too general and doe not imply any connection to perceiver, but if you wish so :rabbit:

Reply:
```suggestion
name: perceiver-io
```